nuxeo-core/nuxeo-core-convert-api/src/main/java/org/nuxeo/ecm/core/convert/api/ConversionService.java

```diff
@@ -36,14 +36,53 @@
 public interface ConversionService {
 
     /**
-     * Gets the convertName given a source and destination MimeType.
+     * Returns the last registered converter name for the given {@code sourceMimeType} and {@code destinationMimeType}.
```

Comment:
"last registered" is an implementation detail of the extension point registry mechanism so I wouldn't mention it here. Just `Returns the converter name...`